### PR TITLE
Config page

### PR DESCRIPTION
**Pull request by zelytra**
Add configuration page. It contains inputs for user configurations and credits of the app
Can now : Change username, backend server hostname and app lang

### CODE REVIEW

File: frontend/src/objects/Fleet.ts [hunk 1]
```diff
@@ -1,118 +1,138 @@
...
+    // Send player data to backend for initialization
+    this.socket.onopen = () => {
+      if (!this.socket) return;
+      this.socket.send(JSON.stringify(UserStore.player));
+    };
+
+    this.socket.onmessage = (ev: MessageEvent<string>) => {
+      const receivedFleet: FleetInterface = JSON.parse(
+        ev.data,
+      ) as FleetInterface; //TODO inspect
```
Comment (dadodasyra) [inline frontend/src/objects/Fleet.ts:53]: Forgotten todo ?

File: frontend/src/vue/alert/Alert.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      this.sendAlert(AlertUtils.getErrorAlert());
+    }
+
+    if (status == 400) {
+      this.sendAlert(AlertUtils.getBadRequest());
+    }
+
+    if (status == 415) {
+      this.sendAlert(AlertUtils.getUnsupportedMediaType());
+    }
```
Comment (dadodasyra) [inline frontend/src/vue/alert/Alert.ts:63]: match case...

**Review by dadodasyra — CHANGES_REQUESTED** [review]
The name of the Pr doesnt include every changes

File: frontend/src/objects/Fleet.ts (hunk 1 above)
Comment (zelytra) [reply frontend/src/objects/Fleet.ts:53]: Patch in future PR

**Review by dadodasyra — APPROVED** [review]

**Pull request merged**